packages/block-library/src/post-featured-image/dimension-controls.js

```diff
@@ -80,6 +75,9 @@ const DimensionControls = ( {
 			label: name,
 		} ) );
 
+	// Get the block Supports aspect ratio.
+	const aspectRatio = attributes?.style?.dimensions?.aspectRatio;
```

Comment:
```
const aspectRatio = attributes?.style?.dimensions?.aspectRatio;
```

Above line can be changed to something like,

```
const blockSupportAspectRatio = attributes?.style?.dimensions?.aspectRatio || aspectRatio;
```
Here we should not remove the attribute from the block itself, we just keep it to get the value which was stored previously? 

But it would have issue like, `blockSupport` aspectRatio dropdown would not show the selected aspectRatio from the atttribute.